prisma-schema/Readme.md

```diff
@@ -814,17 +815,64 @@ model Customer {}
 Type definitions can be used to consolidate various type specifications into one type.
 
 ```groovy
-type Numeric Float @pg.numeric(precision: 5, scale: 2)
-                   @ms.decimal(precision: 5, scale: 2)
+type Numeric {
+  precision Int @bound(gte: 1, lte: 131072)
+  scale Int @bound(gte: 0, lte: 16383)
+}
 
 model User {
   id       Int      @id
-  weight   Numeric
+  weight   Numeric(precision: 5, scale: 20)
 }
 ```
 
 You can attach any field attribute to a type definition.
 
+### Type Definitions provided by Connectors
+
+Connectors can bring their own type definitions allowing you to use these types in your own schemas.
+
+**postgres.prisma (generated by the connector)**
+
+```groovy
+type SmallInt Int @raw("smallint") @bound(gte: -32768, lte: 32767)
```

Comment:
FYI, I changed `@constraint` to `@bound` for no other reason than bound is shorter